src/routes/auth/create-new-user.e2e.test.js

```diff
@@ -0,0 +1,191 @@
+
+describe('POST /auth/create-user', () => {
+	const validPassword = 'validPaswword123$'
+	describe('Failure', () => {
+		let signupInfo
+		let inviteToken
+
+		before(async () => {
+			signupInfo = {
+				firstName: faker.name.firstName(),
+				lastName: faker.name.lastName(),
+				email: faker.internet.email(),
+				password: faker.internet.password()
+			}
+			inviteToken = fixtures.generateInviteToken(signupInfo.email)
+			await fixtures.insertUserInvite(
+				{email: signupInfo.email})
+		})
+
+		it('should throw an error if email is invalid', async () => {
+
+			const expectedError = {
+				"error": {
+					"message": "email must be a valid email"
+				},
+				"status": "failed"
+			}
+			return fixtures.api()
+				.post('/api/v1/auth/create-user')
+				.set('Authorization', `Bearer ${inviteToken}`)
+				.send({
+					firstName: signupInfo.firstName,
+					lastName: signupInfo.lastName,
+					email: 'invalid-email.com',
+					password: validPassword
+				})
+				.expect(400, expectedError)
+		})
+
+		it('should throw an error if password is invalid', async () => {
+			const expectedError = {
+				"error": {
+					"message":
+                     // eslint-disable-next-line max-len
+                     "password length must be at least 8 characters long, Password should contain special characters, Password should contain alphanumeric characters, Password should contain \n\t\t\tuppercase and lowercase characters"
+				},
+				"status": "failed"
+			}
+
+			return fixtures.api()
+				.post('/api/v1/auth/create-user')
+				.set('Authorization', `Bearer ${inviteToken}`)
+				.send({
+					firstName: signupInfo.firstName,
+					lastName: signupInfo.lastName,
+					email: signupInfo.email,
+					password: 'invalid'
+				})
+				.expect(400, expectedError)
+		})
+
+		it('should throw a 401 error if token is not provided', async () => {
+			const expectedError = {
+				"error": {
+					"message": "No token provided."
+				},
+				"status": "failed"
+			}
+
+			return fixtures.api()
+				.post('/api/v1/auth/create-user')
+				.send({
+					firstName: signupInfo.firstName,
+					lastName: signupInfo.lastName,
+					email: signupInfo.email,
+					password: validPassword
+				})
+				.expect(401)
+				.then(res => {
+					expect(res.body).to.eql(expectedError)
+				})
+		})
+
+		it('should throw a 401 error if token is invalid', async () => {
+			const expectedError = {
+				"error": {
+					"message": "Invalid token"
+				},
+				"status": "failed"
+			}
+
+			return fixtures.api()
+				.post('/api/v1/auth/create-user')
+				.set('Authorization', `Bearer ${faker.datatype.uuid()}`)
+				.send({
+					firstName: signupInfo.firstName,
+					lastName: signupInfo.lastName,
+					email: signupInfo.email,
+					password: validPassword
+				})
+				.expect(401)
+				.then(res => {
+					expect(res.body).to.eql(expectedError)
+				})
+		})
+
+		it('should throw a 403 error if token and body emails are different',
```

Comment:
should `return` not throw and kindly let the test description be less technical and not about the code 

for instance `body` here is vague and can't be understood without looking at code